engine/src/test/java/org/camunda/bpm/engine/test/api/externaltask/ExternalTaskServiceTest.java

```diff
@@ -103,6 +103,7 @@ public void setUp() throws Exception {
   @After
   public void tearDown() throws Exception {
     ClockUtil.reset();
+    processEngineConfiguration.setSkipOutputMappingOnCanceledActivities(false);
```

Comment:
I think this will not be necessary anymore as the tests have been moved. 😃

Reply:
damn it! :D

Reply:
I left the other changes to the test class although they are not related to the ticket.

Reply:
Yeah, that is absolutely fine, improving test classes is always nice 👍